test/Android/androidtest.cmd

```diff
@@ -34,6 +35,14 @@ set _GlassDir=%_ProjectRoot%%_GlassPackageName%\
 if NOT exist "%_GlassDir%glass2.exe" echo Getting Glass from NuGet.& call "%_ProjectRoot%tools\NuGet\nuget.exe" install %_GlassPackageName% -Version %_GlassPackageVersion% -ExcludeVersion -OutputDirectory %_ProjectRoot%
 if NOT "%ERRORLEVEL%"=="0" echo ERROR: Failed to get Glass from NuGet.& exit /b -1
 
+:: Copy binaries to folder "Microsoft.VisualStudio.Glass" which required by glass2.exe in runtime for Visual Studio 2017
+if defined VS150COMNTOOLS (
+xcopy /Y /D "%VSINSTALLDIR%Common7\IDE\Remote Debugger\x86\Microsoft.VisualStudio.Debugger.Interop.15.0.dll" "%_GlassDir%"
```

Comment:
glass2 will throw exception "Unable to load one or more of the requested types."  in following code when initialized:

            Type t = typeof(GlassCommand);
            Type[] rgtypes = t.Module.GetTypes();

By checking the $exception.LoaderExceptions, you can see it relied on "Microsoft.VisualStudio.Debugger.Interop.15.0" in Dev15, is it expected ?